client/devdocs/design/index.jsx

```diff
@@ -94,75 +120,82 @@ let DesignAssets = React.createClass( {
 						analyticsGroup="Docs">
 					</SearchCard>
 				}
-				<Collection component={ this.props.component } filter={ this.state.filter }>
-					<Accordions componentUsageStats={ componentsUsageStats.accordion } />
-					<BulkSelect />
-					<ButtonGroups />
-					<Buttons componentUsageStats={ componentsUsageStats.button } />
-					<Cards />
-					<ClipboardButtonInput />
-					<ClipboardButtons />
-					<Count />
-					<CountedTextareas />
-					<DatePicker />
-					<DropZones searchKeywords="drag" />
-					<ExternalLink />
-					<FeatureGate />
-					<FilePickers />
-					<FoldableCard />
-					<FormFields searchKeywords="input textbox textarea radio"/>
-					<Gauge />
-					<GlobalNotices />
-					<Gridicons />
-					<Headers />
-					<InfoPopover />
-					<InputChrono />
-					<Notices />
-					<PaymentLogo />
-					<Popovers />
-					<ProgressBar />
-					<Ranges />
-					<Rating />
-					<SearchDemo />
-					<SectionHeader />
-					<SectionNav />
-					<SegmentedControl />
-					<SelectDropdown searchKeywords="menu" />
-					<SocialLogos />
-					<Spinner searchKeywords="loading" />
-					<SpinnerLine searchKeywords="loading" />
-					<TimezoneDropdown />
-					<TokenFields />
-					<Version />
+				<Collection
+					component={ this.props.component }
+					filter={ this.state.filter }
+				>
+					{
+						[
+							<Accordions />,
+							<BulkSelect />,
+							<ButtonGroups />,
+							<Buttons />,
+							<Cards />,
+							<ClipboardButtonInput />,
+							<ClipboardButtons />,
+							<Count />,
+							<CountedTextareas />,
+							<DatePicker />,
+							<DropZones searchKeywords="drag" />,
+							<ExternalLink />,
+							<FeatureGate />,
+							<FilePickers />,
+							<FoldableCard />,
+							<FormFields searchKeywords="input textbox textarea radio" />,
+							<Gauge />,
+							<GlobalNotices />,
+							<Gridicons />,
+							<Headers />,
+							<InfoPopover />,
+							<InputChrono />,
+							<Notices />,
+							<PaymentLogo />,
+							<Popovers />,
+							<ProgressBar />,
+							<Ranges />,
+							<Rating />,
+							<SearchDemo />,
+							<SectionHeader />,
+							<SectionNav />,
+							<SegmentedControl />,
+							<SelectDropdown searchKeywords="menu" />,
+							<SocialLogos />,
+							<Spinner searchKeywords="loading" />,
+							<SpinnerLine searchKeywords="loading" />,
+							<TimezoneDropdown />,
+							<TokenFields />,
+							<Version />
+						].map( ( Component, idx ) => (
+							React.cloneElement(
+								Component,
+								{
+									getUsageStats,
+									key: `devdocs-design-component-${idx}`
```

Comment:
this is wrong, the key should be unique and preserved overtime i.e. don't use the loop `idx`